fe1-web/src/features/social/screens/SocialHome.tsx

```diff
@@ -79,8 +82,7 @@ const SocialHome = () => {
             value={inputChirp}
             onChangeText={setInputChirp}
             onPress={publishChirp}
-            // The publish button is disabled when the user public key is not defined
-            publishIsDisabledCond={!currentUserPopTokenPublicKey}
+            publishIsDisabledCond={publishDisabled}
```

Comment:
that's a weird attribute name, which causes an even weirder logic.
It might make sense to flip it to `publishEnabled` or at least rename it as `publishDisabled`

Reply:
Yes, I thought that too! Most definitely was on my todo list for refactoring haha :)